compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -515,7 +515,7 @@ int entry(int argc, char **argv)
     luci::change_outputs(graph, new_outputs);
   }
 
-  // call luci optimizations for module
+  // call luci optimizations for module before optimizations for graph
```

Comment:
why is this comment changed to "before..." ?

Reply:
Just to emphasize that after graph optimization there will be another optimization of the entire module. If it only makes it more confusing, I can remove this comment :)

Reply:
comment can give misunderstanding that one "MUST..."
if not "MUST", please restore as before.

Reply:
Done